sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -705,6 +705,7 @@ def _apply_sdk_environment_overrides(
             pattern, override, docker_payload.container_image)
```

Comment:
Yeah, I think this is a bug.

Though I think a better fix would be to change this line to following, so that we continue apply all the overrides in order.

        new_container_image = re.sub(
            pattern, override, new_container_image)

It will be very good to add a unit test for multiple overrides case as well. We have a test for single override case here so you can just add new test based on that.
https://github.com/apache/beam/blob/a8cd05932bed9b2480316fb8518409636cb2733b/sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py#L221

Reply:
Created https://github.com/apache/beam/pull/14579